Improve documentation of makeWrapResolversPlugin (#121)
graphile/graphile.github.io: src/pages/postgraphile/make-wrap-resolvers-plugin.md

**NOTE: this documentation applies to PostGraphile v4.1.0+**

Resolver wrapping enables you to easily take actions before or after an existing GraphQL field resolver, or even to prevent the resolver being called.

> `makeWrapResolversPlugin` is a helper function for you to easily generate your own custom plugin (as a JS module) for 'wrapping' the resolvers generated by PostGraphile. **If using PostGraphile CLI, the generated plugin module should then be provided as an argument to `--append-plugins`, or included in the `appendPlugins` array option when using PostGraphile as a library.**

Reviewer: ```suggestion
`makeWrapResolversPlugin` helps you to easily generate a ["schema plugin"](/postgraphile/extending/) for 'wrapping' the resolvers generated by PostGraphile. You can [load the resulting schema plugin](https://www.graphile.org/postgraphile/extending/#loading-plugins) with `--append-plugins` via the PostGraphile CLI, or with `appendPlugins` via the PostGraphile library.
```

);
```

### Usage

> As mentioned above, the generated plugin module should then be provided as an argument to `--append-plugins` when using PostGraphile CLI, or included in the `appendPlugins` array option when using PostGraphile as a library.

Reviewer: ```suggestion
As mentioned above, you can [load the resulting schema plugin](https://www.graphile.org/postgraphile/extending/#loading-plugins) with `--append-plugins` via the PostGraphile CLI, or with `appendPlugins` via the PostGraphile library.
```

> The above examples defined single plugin object generated by calling `makeWrapResolversPlugin` and exported as the only element in a Javascript file (module). This is equivalent to the following in ES6:

Reviewer: ```suggestion
The above examples defined a single plugin function generated by calling `makeWrapResolversPlugin` and exported via CommonJS as the only element in the JavaScript file (module).

If you are using ES6 modules (`import`/`export`) rather than Common JS (`require`/`exports`), then the syntax should be adjusted slightly:
```

> ```

Reviewer: ```suggestion
```js
```
